docs/index.md

```diff
@@ -12,7 +12,7 @@ Coming soon!
 
 # Features
 
-Coming soon!
+- [TCP health probe](features/tcp-health-probe) allow the runtime to periodically check liveness/readiness of the host.
```

Comment:
How about this?
```suggestion
- Support for exposing TCP health probes to periodically check liveness/readiness of the host ([docs](features/tcp-health-probe))
```

Reply:
Ah, yes; was thinking about the approach used in the other repo's.